### ISSUE TITLE
Allow to config parent mailer

### ISSUE THREAD

**Issue by Uysim** [mikker/passwordless#78]
if we do like this
```ruby
Password.setup do |config|
   config.parent_mailer = "Application"
end
```
And our mailer inherit from that parent_mailer
So we can allow user to custom options and layout by their base on rails config
We can also remove this remove this `Passwordless.default_from_address = "CHANGE_ME@example.com"`

It got this concept from devise. If you are ok I can help

**Comment by mikker** [comment]
This sounds like an okay approach. PR welcome.

We'll need to support both ways until next major (1.0) so no need to pull out the current approach in you PR 😊

**Comment by mikker** [comment]
Closed in #82
